### DISCUSSION TITLE
Failed to load ammo.wasm.js

### DISCUSSION THREAD

**Discussion by TheWizDotBiz** [enable3d/enable3d#371] [Help / Questions] [answered]
![image](https://github.com/enable3d/enable3d/assets/121569215/d6dd4367-d1a3-47c6-9af9-abaefbe2104a)

Hi, currently developing a small online 3d game in three.js with socket.io for a student project.

I've been trying to plug in enable3d to add some basic collision so the local player can't move through walls and may be able to go up and down elevation.

I've been getting this issue for days and no matter how I import or link the ammo libraries or enable3d it always returns the same issue.

The current branch of the project is public, feel free to take a look if this isn't enough info. I've experience making online games in unity but I'm pretty much new to three.js and socket.io networking. Enable3D is being a massive hurdle rn.

[https://github.com/TheWizDotBiz/CART351-TROMPRE/tree/buggyBranch](url)

Thanks!.

**Comment by yandeu** [comment, marked as answer]
I guess your `PhysicsLoader('lib/ammo/kripken/ammo.js')` should be `PhysicsLoader('lib/ammo/kripken')`.

**Reply by TheWizDotBiz** [reply]
Oh man i feel dumb for not trying this out. Anyway it fixed that issue. Thanks a bunch!
